lib/constants/enums.ts

```diff
@@ -180,3 +180,13 @@ export enum DisconnectionReason {
   AuthFailureInvalidSignature = 12,
   WireProtocolErr = 13,
 }
+
+export enum AlertType {
+  LowChannelBalance = 0,
+  LowTradingBalance = 1,
+}
+
+export enum BalanceSide {
```

Comment:
I still think `ChannelSide` makes more sense here than `BalanceSide` and would be reusable if we're ever referring to the side of a channel in a context other than balances.

Also, I don't think we need to explictly write `= 0` or `= `1` here since these aren't going to change, but we should keep it for AlertType since those may change in the future.